NSS-Day-Cohort-53/tabloidcli-bubbling-bullfrogs
# Add note manager

#### Changes Made
1.Added NoteManager Menu to Post Details and Added Go Back to Switch Case
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout Sh-AddNoteManager
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
  
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
